src/objects/npe03__Recurring_Donations_Settings__c.object

```diff
@@ -14,6 +14,16 @@
         <type>Number</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>DisableRollupsWhenCreatingInstallments__c</fullName>
```

Comment:
@sam-knox Would you mind reviewing the label and help text for this new settings field?

Reply:
Thanks @sam-knox. Changes made.